### DISCUSSION TITLE
All my projects have been transferred to a new team that I own automatically

### DISCUSSION THREAD

**Discussion by teleyinex** [vercel/community#6401] [Help]
### Summary

Today, all my projects have been transferred from my account to a team's account that I own. Could this be a process from Vercel to migrate my account to this team thing? I have deleted all the tokens I hactive tokensrrent token for today's login. Any help on this would be really helpful.

### Steps to Reproduce

Just got an  email from Vercel telling me that a project was transferred and I have entered the site to check it.

**Comment by amyegan** [comment]
Hey @teleyinex. It sounds like your account was just updated to our new account structure. We are not charging anyone for this new Hobby team. It simply makes the upgrade process easier if you choose that option in the future.

You can learn more about the change in https://github.com/vercel/community/discussions/6298
